common/autoware_auto_perception_rviz_plugin/src/object_detection/predicted_objects_display.cpp

```diff
@@ -54,8 +54,8 @@ void PredictedObjectsDisplay::processMessage(PredictedObjects::ConstSharedPtr ms
 
     // Get marker for id
     geometry_msgs::msg::Point uuid_vis_position;
-    uuid_vis_position.x = object.kinematics.initial_pose_with_covariance.pose.position.x - 0.5;
-    uuid_vis_position.y = object.kinematics.initial_pose_with_covariance.pose.position.y;
+    uuid_vis_position.x = object.kinematics.initial_pose_with_covariance.pose.position.x;
+    uuid_vis_position.y = object.kinematics.initial_pose_with_covariance.pose.position.y - 0.5;
```

Comment:
@scepter914 Can you share the image taken from view point of a third person?
I think it's good from BEV, but from view point of a third person it would be out of alignment.

Reply:
It may be better to merge into one string and publish